### ISSUE TITLE
MacOS Microphone Access

### ISSUE THREAD

**Issue by ImpossibleReality** [Eugeny/tabby#5672]
**Describe the problem**:
When using a script that uses the microphone in MacOS, there is no prompt to allow microphone access in Tabby (and Tabby is not present in privacy settings), and the script fails.

**To Reproduce**:
* Open a new terminal in Tabby
* Run `ffmpeg -f avfoundation -list_devices true -i ""` to list devices that are available
* Run `ffmpeg -f avfoundation -i ":0" -t 10 test.mp3` to test the microphone (replace `:0` with a colon followed by the microphone device's id from above step)
* Zsh aborts and ffmpeg seems to fail, notice how there is not a prompt to allow microphone access
---
* Open Apple's terminal
* Run above commands
* Observe how there is a prompt to allow microphone access
* It works!

**Comment by nkoane** [comment]
I have the same problem, on Sonoma.

**Comment by KoStard** [comment]
Same issue here, is this prioritized? Are there workarounds?

**Comment by GDKeller** [comment]
Bump for this request, working on speech-to-text not possible without microphone access.

**Comment by jamtur01** [comment]
Interesting - the fix being [the hardened entitlement](https://developer.apple.com/documentation/bundleresources/entitlements/com.apple.security.device.audio-input).

**Comment by Eugeny** [comment]
Yes - the original entitlement was wrong all along as it was the one used for sandboxed apps which tabby isn't

**Comment by jonbaiernx** [comment]
This is still an issue for me on 10.0.230. Is there any fix to try? I have done the usual tccutil resets, but nothing works
